custom-vu/src/main/kotlin/jces1209/vu/action/SwitchBetweenIssuesInSearchResultsCloud.kt

```diff
@@ -0,0 +1,26 @@
+package jces1209.vu.action
+
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.Action
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import jces1209.vu.page.CloudIssueNavigator
+
+class SwitchBetweenIssuesInSearchResultsCloud(
```

Comment:
1. It's to detailed for action. Please make like WorkOnSearch
2. SwitchBetweenIssuesInSearchResultsCloud and SwitchBetweenIssuesInSearchResultsDc are the same
Make one class but specify different pages for cloud and dc

Reply:
or even it's redundant, you can use SearchCloudFilter and SearchServerFilter actions